web-extensions/chrome/engine/datamodel.js

```diff
@@ -435,17 +435,27 @@ BmcDataAPI.prototype.updateComic = function(comicId, chapter, page, cb) {
          * once. We may actually want to accept that, though.
          * Similarly, we may want to allow backwards-tracking too.. ?
          */
-        if (payload.tracking.chapter > chapter
-            || (payload.tracking.chapter == chapter
-                && (payload.tracking.page && page
-                    && payload.tracking.page > page))) {
-            return cb(new Error(LOCALIZATION.getString('S22')));
+        if (payload.tracking.chapter !== undefined && payload.tracking.page !== undefined) {
+            const trackChapter = parseInt(payload.tracking.chapter);
+            const trackPage = parseInt(payload.tracking.page);
+            const currentChapter = parseInt(chapter);
+            const currentPage = parseInt(page);
+
+            if (trackChapter > currentChapter || trackPage > currentPage) {
+                LOGS.warn('S22');
+                // Re-tracking the last tracked page; Reload ? OR clicked on last read ?
```

Comment:
I don't get why this line of comment was pasted here: This is not the case (we check '>', and not `==`
Here, we've attempted to update the tracking with an "older"/"previous" page. (hence the warn just before).

That being said, the next line is still valid.